src/modules/domains/domains.service.ts

```diff
@@ -75,27 +78,20 @@ export class DomainsService {
       ensRegistryAddress,
       provider,
     });
-    ensPublicResolverAddress &&
+      ensResolverV2Address &&
       this._domainDefinitionReader.addKnownResolver({
         chainId,
-        address: ensPublicResolverAddress,
-        type: ResolverContractType.PublicResolver,
-      });
-    ensResolverAddress &&
-      this._domainDefinitionReader.addKnownResolver({
-        chainId,
-        address: ensResolverAddress,
-        type: ResolverContractType.RoleDefinitionResolver_v1,
+        address: ensResolverV2Address,
+        type: ResolverContractType.RoleDefinitionResolver_v2,
       });
```

Comment:
I'm not sure we want to remove the `PublicResolver` code yet as we have yet executed the script. I think I would do this in 3 steps:
1. Use V2 Resolver for Volta
2. Update all of the old domain defs on Volta to use the V2 Resolver
3. Remove the PublicResolver code from ICL (as you are doing in this PR)

What are you thoughts on this approach?